docs/source/additional/releases.yaml

```diff
@@ -9,11 +9,49 @@ components: # for table headers
   - substra-tests
 
 releases:
+  - version: 0.31.0
+    components:
+      substrafl:
+        version: 0.40.0
+        link: https://github.com/Substra/substrafl/releases/tag/0.40.0
+      substra:
+        version: 0.47.0
+        link: https://github.com/Substra/substra/releases/tag/0.47.0
+      substra-tools:
+        version: 0.20.0
+        link: https://github.com/Substra/substra-tools/releases/tag/0.20.0
+      substra-backend:
+        version: 0.41.0
+        link: https://github.com/Substra/substra-backend/releases/tag/0.41.0
+        helm:
+          version: 22.8.1
+          link: https://artifacthub.io/packages/helm/substra/substra-backend/22.8.1
+      orchestrator:
+        version: 0.36.0
+        link: https://github.com/Substra/orchestrator/releases/tag/0.36.0
+        helm:
+          version: 7.5.4
+          link: https://artifacthub.io/packages/helm/substra/orchestrator/7.5.4
+      substra-frontend:
+        version: 0.45.0
+        link: https://github.com/Substra/substra-frontend/releases/tag/0.45.0
+        helm:
+          version: 1.0.23
+          link: https://artifacthub.io/packages/helm/substra/substra-frontend/1.0.23
+      hlf-k8s:
+        version: 0.2.4
+        link: https://github.com/Substra/hlf-k8s/releases/tag/0.2.4
+        helm:
+          version: 10.2.4
+          link: https://artifacthub.io/packages/helm/substra/hlf-k8s/10.2.4
+      substra-tests:
+        version: 0.44.0
+        link: https://github.com/Substra/substra-tests/releases/tag/0.44.0
   - version: 0.30.0
     components:
       substrafl:
-        version: 0.39.0
-        link: https://github.com/Substra/substrafl/releases/tag/0.39.0
+        version: 0.39.1
+        link: https://github.com/Substra/substrafl/releases/tag/0.39.1
```

Comment:
I don't think we should be changing what existing version numbers mean. If we want to integrate these patches let's designate it `0.30.1` -- but really I think a `0.31.0` is sufficient, who would upgrade to `0.30.1` but not `0.31.0`?

Reply:
But that's rewriting history. We're not going to tell people "Oh, are you using the _old_ 0.30.0 or the _new_ 0.30.0?" -- getting rid of this confusion is the point of using version numbers. IMO let's just not bother, but I suppose we could also release a 0.30.1.

Reply:
That's true but should also have 0.29.1 & 0.28.1 then and a bunch other fixes which we never officially made a bundle release for. Let's not bother with this debate rn, i'll put it back to what i did first.

Reply:
Mmmmh I get what you mean @oleobal and you're right... can't wait to the merge of all these repo to have ONE version for all...